src/main/java/com/coffee/gifu/service/OfferService.java

```diff
@@ -2,6 +2,7 @@
 
 import com.coffee.gifu.service.dto.OfferDTO;
 import com.coffee.gifu.service.exception.ManagementRulesException;
+import org.springframework.transaction.annotation.Transactional;
```

Comment:
**Remove this unused import 'org.springframework.transaction.annotation.Transactional'.**